codes/bits/zetterberg.yml

```diff
@@ -0,0 +1,37 @@
+
+code_id: zetterberg
+physical: bits
+logical: bits
+
+name: 'Zetterberg code'
+introduced: '\cite{doi:10.1109/TIT.1962.1057682}'
+
+
+description: 'The Zetterberg code \(C_s\) is a binary cyclic code of length \(n = 2^{2s}+1\) generated by the
+minimal polynomial \(g_s(x)\) of \(\alpha\) over GF(2), where \(\alpha\) is a primitive \(n^{th}\) root of unity in the field GF\((2^{4s})\).'
+
+protection: 'Correct all weight-2 errors with a small number of exceptions. By making some alterations to the code, one can form codes that correct all weight-2 errors.'

```

Comment:
The code as described has distance 5, and it therefore corrects all weight-2 errors with no exceptions. Not sure which alterations are required.